apps/server/src/modules/files-storage/helper/promise.ts

```diff
@@ -0,0 +1,10 @@
+export function isFulfilled<T>(input: PromiseSettledResult<T>): input is PromiseFulfilledResult<T> {
+	return input.status === 'fulfilled';
+}
+
+export function getResolvedValues<T>(result: PromiseSettledResult<T>[]): T[] {
+	const resolvedPromises: PromiseFulfilledResult<T>[] = result.filter(isFulfilled);
+	const fileResponses = resolvedPromises.map((item) => item.value);
```

Comment:
`getResolvedValues` sounds generic but `fileResponses` rather specific. Not knowing the context of usage, this is irritating to me.

Reply:
Missed that naming when making a generic function out of it. Changed it to `const values`.